source/read/change-streams.txt

```diff
@@ -0,0 +1,272 @@
+.. _scala-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your data. A change stream is a {+mdb-server+} feature that
+allows your application to subscribe to data changes on a collection, database,
+or deployment.
+
+When using the {+driver-short+}, you can call the ``watch()`` method to return an
+instance of ``ChangeStreamObservable``. Then, you can subscribe to the
+``ChangeStreamObservable`` instance to see new data changes, such as updates,
+insertions, and deletions.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``restaurants`` collection in the ``sample_restaurants``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To access this collection
+from your Scala application, create a ``MongoClient`` that connects to an Atlas cluster
+and assign the following values to your ``database`` and ``collection`` variables:
+
+.. literalinclude:: /includes/read/change-streams.scala
+    :language: scala
+    :dedent:
+    :start-after: start-db-coll
+    :end-before: end-db-coll
+
+.. tip::
+
+   To learn how to create a free MongoDB Atlas cluster and load the sample datasets,
+   see the :atlas:`Get Started with Atlas </getting-started>` guide.
+
+Some examples use instances of the ``LatchedObserver`` class to handle change
+stream events. This class is a custom observer that prints change stream events
+and continues monitoring data changes until the stream completes or generates
+an error. To use the ``LatchedObserver`` class, paste the following code into
+your application file:
+
+.. literalinclude:: /includes/read/change-streams.scala
+    :language: scala
+    :dedent:
+    :start-after: start-latched-observer
+    :end-before: end-latched-observer
+
+Open a Change Stream
+--------------------
+
+To open a change stream, call the ``watch()`` method. The instance on which you
+call the ``watch()`` method determines the scope of events that the change
+stream monitors. You can call the ``watch()`` method on instances of the following
+classes:
+
+- ``MongoClient``: Monitor all changes in the MongoDB deployment
```

Comment:
Q: does this mean all changes across all collections in all databases? if so, probably worth being explicit about that. i'm thinking there are a lot of config changes you could make to a deployment, like sharding and whatnot

Reply:
Yeah changes to all non-system collections across all databases, according to the Server docs. I'll reword to make that clear